echo/echo-pubsub-aws/src/main/java/com/netflix/spinnaker/echo/pubsub/aws/AmazonMessageAcknowledger.java

```diff
@@ -53,17 +54,15 @@ public AmazonMessageAcknowledger(
 
   @Override
   public void ack() {
-    // Delete from queue
     try {
-      amazonSQS.deleteMessage(queueUrl, message.getReceiptHandle());
+      sqsClient.deleteMessage(
+          DeleteMessageRequest.builder()
+              .queueUrl(queueUrl)
+              .receiptHandle(message.receiptHandle())
+              .build());
       registry.counter(getProcessedMetricId(subscriptionName)).increment();
-    } catch (ReceiptHandleIsInvalidException e) {
-      log.warn(
-          "Error deleting message: {}, queue: {}, reason: {} (receiptHandle: {})",
-          message.getMessageId(),
-          queueUrl,
-          e.getMessage(),
-          message.getReceiptHandle());
+    } catch (SqsException e) {
+      log.warn("Failed to delete message from queue {}: {}", queueUrl, e.getMessage());
```

Comment:
Possible to reduce diffs in the message?  I see SqsException has a requestId property.  Is that the same as `message.getMessageId()` ?  If the receipt handle was helpful before, I figure it's still helpful.  I don't see that in SqsException.